### DISCUSSION TITLE
'Working with Templates' docs page incomplete

### DISCUSSION THREAD

**Discussion by andrewmackie** [aminalaee/sqladmin#318] [General]
Thanks for this repo, Amin.

The [Working with Templates](https://aminalaee.dev/sqladmin/working_with_templates/) docs page ends abruptly - could you please write the section on customizing templates?

**Comment by aminalaee** [comment]
Hello @andrewmackie ,
What do you think is needed there? PRs are also welcome to improve it.

**Reply by Mooreel** [reply]
What would be great are examples for creating Custom Views, how to access data etc.
The page ends with "Customizing Views" as a title.

**Reply by aminalaee** [reply]
I will try to think of something, but I think custom views are different from customizing the templates.
You can find the custom view example here: https://aminalaee.dev/sqladmin/configurations/#custom-views but yeah full examples are not there.

**Reply by andrewmackie** [reply]
Thanks for your reply, sorry that I missed it last year.

In any case, I don't know what is needed in the documentation because I don't know what is possible with respect to customizing views ... I was hoping that the documentation would tell me that! I agree with @Mooreel that some examples would be helpful, wherever it is appropriate to put them.

**Reply by aminalaee** [reply]
I've added custom views here: https://aminalaee.dev/sqladmin/writing_custom_views/
I will do the same for templates :)

**Reply by kenmoh** [reply]
I customize my admin dashboard template by creating partials. I can do a documentation on it if it will help. 
I wanted my dashboard to show the logged in user name and profile image.

**Reply by aminalaee** [reply]
Yeah sure, if you feel like you can improve it go ahead.
